server/data/quotes.json

```diff
@@ -607,7 +611,7 @@
     "author": "Stephen King",
     "sentence": "French is the language that turns dirt into romance."
   },
-    {
+  {
     "author": "Stewart Stafford",
```

Comment:
Please remove or replace this quote, it has already been merged before.

Reply:
There are other quotes from the same author that have been merged but mine is a new quote.

Reply:
Hey, take a look https://github.com/jayehernandez/letra-extension/blob/1e889accae18d3a948bcd9cf09bc4b1fcb55d6b7/server/data/quotes.json#L611. You see, it's already there in the _quotes.json_ file. Only this quote is a duplicate, your other quote by Ferdinand de Saussure is unique so that can be accepted.

I request you to either remove this quote or add a new unique one instead (please check [this file](https://github.com/jayehernandez/letra-extension/blob/master/server/data/quotes.json) before you add a new quote for any duplicates). Thanks!

Reply:
I didn't add the quote from Stewart Stafford, the modification before that line was just the alignment of the curly bracket. Thanks!

Reply:
Oh, my bad. Sorry for opening this review. Your PR looks good!